.travis.yml

```diff
@@ -29,6 +29,8 @@ script:
   - export RUST_BACKTRACE=1
   - export CARGO_INCREMENTAL=1
   - export PATH="$PATH":~/.cargo/bin
+  - cargo build
+  - cargo build --all-features
```

Comment:
Few things here:

* Build is already implicit/dependent part of test. So calling it separately doesn't bring anything new.
* It would, however, be nice to actually even run the rayon tests too. So it would be nice to have both `test` (without extra features) and (`test --all-features`).
* Unimportant nitpick: my original idea was to have another variable in the test matrix, effectively adding more jobs that can run in parallel. On the other hand, running it in a single job reuses all the dependencies already compiled, so it's hard to know if parallelizing actually brings any benefit. So I'll leave the decision up to you if you want to leave it inside the same job, or if you prefer the parallel version.